lib/tasks/bundlers/generateLibraryPreload.js

```diff
@@ -67,6 +67,93 @@ function getBundleDefinition(namespace) {
 	};
 }
 
+function getSapUiCoreBunDef(name, filters, preload=false) {
+	const bundleDefinition = {
+		name,
+		sections: []
+	};
+
+	// add raw section
+	bundleDefinition.sections.push({
+		// include all 'raw' modules that are needed for the UI5 loader
+		mode: "raw",
+		filters,
+		resolve: true, // dependencies for raw modules are taken from shims in .library files
+		sort: true, // topological sort on raw modules is mandatory
+		declareModules: false
+	});
+
+	if (preload) {
+		// add preload section
+		bundleDefinition.sections.push({
+			mode: "preload",
+			filters: [
+				"sap/ui/core/Core.js"
+			],
+			resolve: true
+		});
+	}
+
+	bundleDefinition.sections.push({
+		mode: "require",
+		filters: [
+			"sap/ui/core/Core.js"
+		]
+	});
+
+	return bundleDefinition;
+}
+
+function getNojQueryBunDef(name, filters, preload=false) {
+	const bundleDefinition = {
+		name,
+		sections: []
+	};
+
+	// add provided section
+	bundleDefinition.sections.push({
```

Comment:
Couldn't you just `unshift` this section to the definition created via `getSapUiCoreBunDef` ?
It seems that this is the only difference.

Reply:
Done.